javascript-pokemon-card-lookup/server.js

```diff
@@ -0,0 +1,31 @@
+require('dotenv').config();
+
+const express = require('express');
+const app = express();
+
+const apiKey = process.env.API_KEY;
+
+// let searchTerm = "venusaur" // temp hardcoded
+
+app.get('/search', async (req, res) => {
+    
+    let searchTerm = req.query.q;
+    console.log("NEW SEARCH STARTED!")
+
+    fetch(`https://api.pokemontcg.io/v2/cards?q=name:${searchTerm}`, {
+        headers: {
+            'X-Api-Key': apiKey
+        }
+    })
+        // .then(response => console.log(response))
+        .then(response => response.json())
```

Comment:
for an async function, try to use `await promise` mechanics instead of `promise.then()`. But it's good to get used to using both patterns